src/stm8s_clk.c

```diff
@@ -45,7 +45,7 @@
   * @{
   */
 
-CONST uint8_t HSIDivFactor[4] = {1, 2, 4, 8}; /*!< Holds the different HSI Divider factors */
+CONST uint8_t HISDivExp[4] = {0, 1, 2, 4}; /*!< Holds the different HSI Divider factors */
```

Comment:
(1) Array name should be HSIDivExp to align to old name (here and on line 562 below).
(2) HSIDivExp should be {0, 1, 2, 3} to correspond to the old array values {1, 2, 4, 8}.
(3) Comment on the line should be updated as well (since these are no longer divider factors).

Reply:
(1) oops
(2) I had a brainfart.
(3) I agree.
I'll push an updated commit in a few seconds, it should appear in the PR.